ci/scripts/shell

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
Could you maybe describe the purpose of the script?

Reply:
Yes. The script was copied from HAproxy where there is a README for acceptance tests that describes it. I could also add a read me to pcap.

Reply:
I cannot find in HAProxy acceptance-tests README or in ci README about this script

Reply:
It is a good point to add a README file which contains how to run the scripts and tests locally.

Reply:
Again, I would like to use the opportunity of us touching all these scripts (even if they're copied from HAProxy) to improve them. A lot of this improvement is documentation. We cannot introduce new technical dept in pcap-release though.

Reply:
I've added acceptance-tests/README.md and a comment to this script that explains how it works and what the intention is.